src/codewhisperer/commands/onAcceptance.ts

```diff
@@ -94,8 +94,11 @@ export async function onAcceptance(acceptanceEntry: OnRecommendationAcceptanceEn
             completionType: acceptanceEntry.completionType,
             language: languageContext.language,
         })
-        CodeWhispererCodeCoverageTracker.getTracker(languageContext.language, globalStorage).setAcceptedTokens(
-            acceptanceEntry.recommendation
+        const codeRangeAfterFormat = new vscode.Range(start, acceptanceEntry.editor.selection.active)
+        CodeWhispererCodeCoverageTracker.getTracker(languageContext.language, globalStorage)?.countAcceptedTokens(
```

Comment:
since `countAcceptedTokens` will also update totalToken, and documentChanged event also will update totalToken, will the calculation be performed twice

Reply:
It won't. We are using `vsCodeState.isCodeWhispererEditing` to filter documentChange event from CWSPR